.changeset/warm-lizards-itch.md

```diff
@@ -0,0 +1,5 @@
+---
+'@lg-chat/rich-links': minor
```

Comment:
It could be a patch change since you're fixing a few edge cases without introducing new functionality.